angular/defs.bzl

```diff
@@ -134,7 +134,10 @@ def ng_app(name, project_name = None, deps = [], test_deps = [], **kwargs):
         name = "test",
         chdir = native.package_name(),
         args = ["%s:test" % project_name],
+        include_declarations = True,
```

Comment:
We need declarations for tests? 🤔

Reply:
Whoops. Shouldn't here. Snuck in from debugging the test failure.